libs/sdk-ui-tests/stories/visual-regression/kit/ShareDialog/AddGranteeBase.tsx

```diff
@@ -11,54 +11,102 @@ import { AddGranteeBase } from "@gooddata/sdk-ui-kit";
 import "@gooddata/sdk-ui-kit/styles/css/main.css";
 import "../styles/goodstrap.scss";
 import { current, group, groupAll, user } from "./GranteeMock";
+import { BackendProvider, WorkspaceProvider } from "@gooddata/sdk-ui";
+import { recordedBackend, RecordedBackendConfig } from "@gooddata/sdk-backend-mockingbird";
+import { ReferenceRecordings } from "@gooddata/reference-workspace";
 
 const EmptySelectionExample = (): JSX.Element => {
+    const config: RecordedBackendConfig = {
+        accessControl: {
+            accessList: [],
+        },
+        users: {
+            users: [],
+        },
+    };
```

Comment:
just curious.. don't know much about this area - but this seems like it should be default config for those mocked APIs?

Reply:
done used default value